pkg/synchronization-controller/synchronization-controller.go

```
@@ -194,19 +195,22 @@ func (s *SynchronizationController) deleteMigrationFunc(delObj interface{}) {
 	}
 }
 
-func (s *SynchronizationController) closeConnectionForMigrationID(syncMap *sync.Map, migrationID string) {
+func (s *SynchronizationController) closeConnectionForMigrationID(syncMap *sync.Map, migrationID string) error {
```

Looks like the error is never checked by anything